portfolio/src/main/webapp/script.js

```diff
@@ -4,21 +4,22 @@
  */
 "use strict";
 
-// Get all buttons. coll is an array of html elements
+
+// Get all collapsibles. coll is an array of html elements
 let coll = document.getElementsByClassName("collapsible");
 let i;
 
 for (i = 0; i < coll.length; i++) {
   // Add onclick functionality to every item in coll
-  coll[i].addEventListener("click", respondToClick);
+  coll[i].addEventListener("click", collapsibleRespondToClick);
 }
 
 /**
  Designed to give HTML elements of the collapsible class the ability to respond to click events.
  Responds to click by toggling the style.display property of the next element sibling in the DOM.
  @param clickEvent a clickEvent object on a collapsible item
  */
-function respondToClick(clickEvent) {
+function collapsibleRespondToClick(clickEvent) {
```

Comment:
This is fine for now, but in the future, since this doesn't pertain to the comments feature, it would be better if this was in another pull request.

Reply:
Got it, I will keep an eye out for that in the future